docs/source/test_network.md

```diff
@@ -656,17 +654,6 @@ If you have any problems with the tutorial, review the following:
    ```
    Select ``y``.
 
--  If you try to create a channel with the command `./network.sh createChannel`,
```

Comment:
why remove this?

Reply:
Look at the resolution on line 667... I didn't think it made sense to recommend Docker Desktop 2.5.0.1 reinstall and reclone fabric-samples. I haven't heard of similar problems in modern history.

Reply:
> Look at the resolution on line 667... I didn't think it made sense to recommend Docker Desktop 2.5.0.1 reinstall and reclone fabric-samples. I haven't heard of similar problems in modern history.

That's because Fabric is ancient history by now!